doc/installation_windows.md

```diff
@@ -1,18 +1,13 @@
-# Windows 8.1 & Windows 10 Installation
+# Windows 10 & Windows 11 Installation
 
 **Note:** Due to the USB 3.0 translation layer between native hardware and virtual machine, the librealsense team does not recommend or support installation in a VM.
 
-librealsense shall be built on Windows using [CMake v3.8+](https://cmake.org/download/) and Visual Studio 2015 / 2017: 
+librealsense shall be built on Windows using [CMake v3.8+](https://cmake.org/download/) and Visual Studio 2015 / 2017 / 2019 / 2022: 
```

Comment:
Please remove 2015

Reply:
Also we know Cmake 3.8 is not supported, we need to make sure which one is supported and list it

Reply:
> Also we know Cmake 3.8 is not supported, we need to make sure which one is supported and list it

According the test that we performed CMake 3.14 is minimal supported version.
Are you think we need perform other test to validate it? Maybe, install CMake 3.13 locally and build?